apps/agent/contracts/test/TestImports.sol

```diff
@@ -1,8 +1,12 @@
 pragma solidity 0.4.24;
 
 import "@aragon/os/contracts/acl/ACL.sol";
+import "@aragon/os/contracts/apm/APMRegistry.sol";
```

Comment:
The new imports are needed to deploy the Aragon bases on the local ganache chain we ran with the buidler plugin.